Index: .gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>wwwroot/*.js\r\nnode_modules\r\ntypings\r\ndist\r\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/.gitignore b/.gitignore
--- a/.gitignore	(revision b4b400c9dce21bacaff77dcfd5d3c3d1cb6d4dff)
+++ b/.gitignore	(date 1703449702337)
@@ -2,3 +2,8 @@
 node_modules
 typings
 dist
+# Angular cache
+.angular/cache/
+
+
+
